src/adapters/telegram/helpers/client.ts

```diff
@@ -29,7 +29,7 @@ export const telegramFormattedNotifier = (messagePayload: TLMessageFormattedPayl
 
     const sendInterval = setInterval(async () => {
       clearInterval(sendInterval);
-      await telegram.telegram.sendMessage(chatIds[currentElem], text, { parse_mode: parseMode });
+      await telegram.sendMessage(chatIds[currentElem], text, { parse_mode: parseMode });
```

Comment:
```
      await telegramClient.sendMessage(chatIds[currentElem], text, { parse_mode: parseMode });
```

Something like this seems pretty clear.

Reply:
we are currently using`supabase` instead of `supabaseClient` so lets keep it like `await telegram.sendMessage(...`